src/osmium/replication/server.py

```diff
@@ -324,7 +320,7 @@ def get_diff_block(self, seq):
             (or :code:`urllib2.HTTPError` in python2)
             if the file cannot be downloaded.
         """
-        return self.open_url(self.make_request(self.get_diff_url(seq))).read()
+        return self.open_url(self.get_diff_url(seq)).content
```

Comment:
Same here with `read()` vs. `content`. But I think that's all we use from the response object.